xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-docker/src/test/it/org/xwiki/administration/test/ui/AdministrationIT.java

```diff
@@ -71,11 +72,12 @@ void verifyAdministrationSections(TestUtils setup)
         assertTrue(wikiAdministrationPage.hasNotSection("PageRights"));
 
         // Select XWiki page administration.
-        setup.gotoPage("NonExistentSpace", "WebHome");
+        setup.gotoPage(testReference);
         page = new AdministrablePage();
         AdministrationPage pageAdministrationPage = page.clickAdministerPage();
-
-        assertEquals("Page Administration: NonExistentSpace", pageAdministrationPage.getDocumentTitle());
+        String fullName = setup.serializeReference(testReference.getParent()).split(":")[1];
```

Comment:
I think what you were searching for is #serializeLocalReference (which does not contain the wiki part).

Reply:
Hmmm I didn't paid enough attention: that one is not available in 15.10.x / 16.4.x. Backporting it.